packages/palette/src/index.tsx

```diff
@@ -3,6 +3,7 @@ export * from "./helpers"
 export * from "./svgs"
 export * from "./Theme"
 export * from "./utils"
+export * from "./themes/types"
```

Comment:
hm?

Reply:
some files need `Color` and `SpacingUnit` types, so I thought I'll export them. you have a better idea?

Reply:
i should clarify, files outside palette. like palette-charts

Reply:
How were they getting them previously?

Reply:
they were coming from the `v2.ts` files that are now gone.
For example, for `Color`:
- https://github.com/artsy/palette/blob/main/packages/palette-tokens/src/themes/v2.tsx#L373
- https://github.com/artsy/palette/blob/main/packages/palette-tokens/src/themes/index.ts#L13
- https://github.com/artsy/palette/blob/main/packages/palette-tokens/src/index.ts#L13

just very "export everything" style, and now these are at least only coming from `themes/types`, which then come from tokens. feels better.

Reply:
But they were coming from the tokens package; where as this is in the parent palette package. I guess it's just less confusing if it's coming from the one

Reply:
Well that's the thing; everything should come from tokens since that's what gets shared. Tokens should actually be used by Eigen, for instance.

Reply:
If it's not then we don't even need it separate. cc @damassi

Reply:
Oh right, and charts uses it too.

Reply:
I would just leave things as they were.

Reply:
shouldnt it be in here then? https://github.com/artsy/palette/blob/main/packages/palette-charts/package.json#L117-L124

Reply:
Hm, probably, I guess since it's a transitive dependency of Palette it doesn't yell.

Reply:
alright. in that case, i can remove this, and let it be imported from tokens 👍

Reply:
i think in that case we should remove this file too then, kinda useless.
```
import {
  Breakpoint as BreakpointV3,
  Color as ColorV3,
  SpacingUnit as SpacingUnitV3,
} from "@artsy/palette-tokens/dist/themes/v3"

export type Color = ColorV3
export type SpacingUnit = SpacingUnitV3
export type Breakpoint = BreakpointV3
```

Reply:
preference on
```
import { SpacingUnit } from "@artsy/palette-tokens"
```
or
```
import { SpacingUnit } from "@artsy/palette-tokens/dist/themes/v3"
```
?